internal/manager/config.go

```diff
@@ -284,6 +287,19 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	return flagSet
 }
 
+func (c *Config) Resolve() error {
```

Comment:
Could we please add a comment for this public function? 

I was also wondering whether we could cover it with a simple unit test reading from a temp file, WDYT?

Reply:
I as thinking it too, and finally did the coverage with the config_validaiton_test.go.

Reply:
IMO it would be better to keep tests for both functions isolated. Testing `Resolve` in the `config_validation_test.go` doesn't seem correct to me as this function is not responsible for validation itself, but for resolving the config values if they're not directly set. Would you mind adding `config_test.go` with `TestConfigResolve` test case that will exclusively test this function?

Reply:
@czeslavo done.

Reply:
Thanks! `internal/manager/config_test.go` fails to be compiled due to missing imports, please take a look.